src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -142,9 +147,11 @@ function getDrupalContent(buildOptions) {
       log(err.stack);
       log(JSON.stringify(drupalData));
       log('Failed to pipe Drupal content into Metalsmith!');
-      log('Continuing with build anyway...');
-      // done(err);
-      done();
+      if (buildOptions.buildtype !== ENVIRONMENTS.LOCALHOST) {
+        done(err);
+      } else {
+        done();
```

Comment:
If we don't do this, then devs who don't care about Drupal content will have to be on the SOCKS proxy. Also, Heroku will stop working, I think.